app/assets/sass/application.scss

```diff
@@ -416,21 +416,30 @@
 
   .govuk-title {
     .context {
-      a {
-        color: $text-colour;
-        text-decoration: underline;
-
-        &:before {
-          content: " ";
-          display: inline-block;
-          height: 0;
-          width: 0;
-          border-top: 10px solid transparent;
-          border-bottom: 10px solid transparent;
-          border-right: 10px solid $black;
-          margin-right: 10px;
+      
+      display: none;
+      
+      @include media(mobile) {
```

Comment:
I don't think this would have any effect – `media(mobile)` [targets everything over 1px wide](https://github.com/alphagov/govuk_frontend_toolkit/blob/2011e77436fccd31f32a0a975a2bf4cc3b961495/stylesheets/_conditionals.scss) – I think you'd want `media(tablet)`.